website/www/site/content/en/documentation/transforms/java/elementwise/regex.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Regex"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Regex
+<table align="left">
+    <a target="_blank" class="button"
+        href="https://beam.apache.org/releases/javadoc/current/index.html?org/apache/beam/sdk/transforms/Regex.html">
+      <img src="https://beam.apache.org/images/logos/sdks/java.png" width="20px" height="20px"
+           alt="Javadoc" />
+     Javadoc
+    </a>
+</table>
+<br><br>
```

Comment:
Why do we multiple breaks here?

Reply:
In Jekyll, there was only one `<br>` at this line and once it was rendered, this `<br>` would be wrapped by `<p><br></p>`. In Hugo, `<br>` is still `<br>` after rendering so the spacing here is different. I don't wanna mimic `<p><br></p>` because it doesn't make sense, so I add one more `<br>` and now they have the same spacing.